ucp/_libs/ucx_api.pyx

```diff
@@ -212,7 +212,11 @@ cdef class UCXContext(UCXObject):
         dict _config
         readonly bint cuda_support
 
-    def __init__(self, config_dict):
+    def __init__(
+        self,
+        config_dict,
+        future_flags=(UCP_FEATURE_TAG | UCP_FEATURE_WAKEUP | UCP_FEATURE_STREAM)
```

Comment:
Did you mean to name this `feature(s)_flags` ?

Reply:
Yes :)